compiler/rustc_codegen_cranelift/example/std_example.rs

```diff
@@ -8,9 +8,6 @@
     unboxed_closures
 )]
 #![allow(internal_features)]
-// FIXME once abi_unsupported_vector_types is a hard error disable the foo test when the respective
-// target feature is not enabled.
```

Comment:
@bjorn3 apparently I have to "disable the foo test when the respective target feature is not enabled". What does that mean?^^

Reply:
There is a function called `foo` which accepts a `I64X2` type. It can only be used when the sse target feature is enabled. Given that cg_clif always enables the sse target feature for non -none targets, I don't think it is necessary to disable it after all. So nothing needs to be changed in this PR.